### PR TITLE
Cleanup result format

### PR DESCRIPTION
**Pull request by papr**

### CODE REVIEW

File: pye3d/detector_3d.py [hunk 1]
```diff
@@ -544,6 +544,8 @@ def _prepare_result(
             projected_pupil_circle = Ellipse(np.asarray([0.0, 0.0]), 0.0, 0.0, 0.0)
 
         result["ellipse"] = ellipse2dict(projected_pupil_circle)
+        result["location"] = result["ellipse"]["center"]  # pupil center in pixels
```
Comment (euryalus) [inline pye3d/detector_3d.py:547]: What is the reason to repeat this information? After all, it is already in result[''ellipse"].

File: pye3d/detector_3d.py [hunk 2]
```diff
@@ -544,6 +544,8 @@ def _prepare_result(
...
+
         result["diameter"] = projected_pupil_circle.major_radius
```
Comment (euryalus) [inline pye3d/detector_3d.py:549]: Same as above. Also, is the major_radius really the diameter? Or half of the diamter?

File: pye3d/detector_3d.py (hunk 1 above)
Comment (papr) [reply pye3d/detector_3d.py:547]: Both 2d and previous 3d detectors included these keys. Also, I feel that both keys (locations and diameter) are very common keys, so including them as a quality-of-life enhancement makes sense to me. I agree that it is redundant information. Therefore, I do not have a strong position on this.

File: pye3d/detector_3d.py (hunk 2 above)
Comment (papr) [reply pye3d/detector_3d.py:549]: @euryalus The naming suggests it is half the diameter. Is there an easy way to check?

File: pye3d/detector_3d.py (hunk 1 above)
Comment (euryalus) [reply pye3d/detector_3d.py:547]: Sounds good.

File: pye3d/detector_3d.py (hunk 2 above)
Comment (euryalus) [reply pye3d/detector_3d.py:549]: Have looked in the code. Here, the major_radius is the diameter.

Comment (papr) [reply pye3d/detector_3d.py:549]:
```suggestion
        # projected_pupil_circle is an OpenCV ellipse, i.e. major_radius is major diameter
        result["diameter"] = projected_pupil_circle.major_radius
```

**Review by euryalus — APPROVED** [review]

**Pull request merged**